test/query.spec.js

```diff
@@ -193,4 +193,49 @@ describe('query.execute()', () => {
         expect(res.body.results.bindings).toHaveLength(1);
       }));
   });
+
+  describe('update', () => {
+    it('should insert some data', () =>
+      execute(`insert data {:foo :bar :baz}`)
+        .then(res => {
+          expect(res.status).toBe(200);
+          expect(res.body).toBe(null);
+          return execute(`select * {:foo ?p ?o}`);
+        })
+        .then(res => {
+          expect(res.body.results.bindings).toHaveLength(1);
+        }));
+
+    it('should remove some data', () =>
```

Comment:
This test is asserting that a triple is not in the database, attempting to delete it twice, and then asserting that it's still not there. I think something got lost in translation